connector/src/main/java/org/geysermc/connector/network/translators/java/entity/JavaEntityStatusTranslator.java

```diff
@@ -91,18 +98,21 @@ public void translate(ServerEntityStatusPacket packet, GeyserSession session) {
             case LIVING_DEATH:
                 entityEventPacket.setType(EntityEventType.DEATH);
                 if (entity.getEntityType() == EntityType.THROWN_EGG) {
+                    ItemEntry eggItemJava = ItemRegistry.getItemEntry("minecraft:egg");
```

Comment:
Wow, I did not realize that's what the number was! Nice catch.

As this number isn't going to change on us, perhaps make it a static constant in `ItemRegistry`? (Match what we do there with the other items)